### ISSUE TITLE
ASwitch component focus

### ISSUE THREAD

**Issue by peterbud** [jd-solanki/anu#29]
I love the components so far!

I have noticed when I was testing that `ASwitch` component cannot not get a keyboard focus. This also means that it cannot be toggled by keyboard, only by mouse, which from the accessibility POV not ideal.

I believe the internal hidden `input[type='checkbox']` is [hidden](https://github.com/jd-solanki/anu/blob/main/packages/anu-vue/src/components/switch/ASwitch.tsx#L44), and that leads to this problem. 

I have seen other frameworks are not hiding this internal input, but making it "invisible" with CSS like: 
```
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  width: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
```

I'm not sure tough that this is the best solution...

As a temporary solution one can apply a similar CSS rule which at least makes it indeed possible to get a keyboard focus on the component, but also a visual clue would be also needed that the component has a keyboard focus.

**Comment by jd-solanki** [comment]
Hi 👋🏻

Thanks for raising the issue. Yes, `ASwitch` currently can't get focused.

I will try this soon and update the code.

Thanks.
